### PR TITLE
Moved adapter factory from in-config to seperate factory class + other small improvements

### PR DESCRIPTION
**Pull request by nepda**
Moved key/salt into local config file (don't inlcude key/salt into your VCS!)
Small code cleanups / small improvements

### CODE REVIEW

**Comment by nepda** [comment]
Please let me know, if you're interested in merging this and releasing a new version (e.g. v1.1.0).

File: config/doctrine-encrypt-module.global.php.dist [hunk 1]
```diff
@@ -8,31 +8,27 @@
...
 
                 //The reader will read encryption configuration information off of the entities
                 //You probably don't have to change this
-                'reader' => 'Doctrine\Common\Annotations\AnnotationReader',
-            )
+                # 'reader' => 'Doctrine\Common\Annotations\AnnotationReader',
+            ),
         ),
 
         'eventmanager' => array(
```
Comment (51systems) [inline config/doctrine-encrypt-module.global.php.dist]: If we move the event manager definition to the module.config.php is there any point in even keeping this file?

Comment (nepda) [inline config/doctrine-encrypt-module.global.php.dist]: I think the file should be available, to give users the possibility to change the adapter and the reader. We could improve the documentation about the lines. Take a look into the ZfcUser config example: https://github.com/ZF-Commons/ZfcUser/blob/1.x/config/zfcuser.global.php.dist

Comment (nepda) [inline config/doctrine-encrypt-module.global.php.dist]: Moving the EventManager to the module.config.php is a nice idea. So the listener is attached, as soon as the module is active. Patch is on the way...

**Comment by nepda** [comment]
I would recommend the release of version v1.1.0 after merge.

**Pull request merged**